package.json

```diff
@@ -14,10 +14,19 @@
   "contributes": {
     "commands": [
       {
-        "command": "annaPrototype.helloWorld",
-        "title": "Hello World"
+        "command": "annaPrototype.showReferrer",
+        "title": "Show Referrer"
       }
-    ]
+    ],
+    "menus": {
+      "view/item/context": [
+        {
+          "command": "annaPrototype.showReferrer",
```

Comment:
annaPrototype is not production code.  Please start taking out any name references to your name or 'prototype'